apps/mobile/src/features/send/send-form/sheets/recipient-sheet.tsx

```diff
@@ -26,7 +26,7 @@ export function RecipientSheet<T extends SendFormBaseContext<T>>({
 
   return (
     <Sheet ref={sheetRef} themeVariant={themeDerivedFromThemePreference}>
-      <Box pt="5" px="5">
+      <Box p="5" gap="1" height={112}>
```

Comment:
Setting the height here means the sheet can open on Android but now there is too much space below it on iOS:

![Screenshot 2025-01-22 at 11 00 21](https://github.com/user-attachments/assets/41c1e243-1a9d-424f-bddd-c5c8c827c843)

I didn't want to start adding platform specific code but maybe that's what we should do here?

This flow is being re-vamped anyway so I can ask the design team about this

Reply:
I think this is fine for the time being - future plans will add more content here so it'll be a non-issue in the midterm.